src/apsbits/core/run_engine_init.py

```diff
@@ -110,12 +117,50 @@ def init_RE(
                 f"without saving metadata to disk. {error=}\n"
             )
 
-    if cat_instance is not None:
-        RE.md.update(re_metadata(iconfig, cat_instance))  # programmatic metadata
-        RE.md.update(re_config.get("DEFAULT_METADATA", {}))
-        RE.subscribe(cat_instance.v1.insert)
-    if bec_instance is not None:
-        RE.subscribe(bec_instance)
+    RE.md.update(re_config.get("DEFAULT_METADATA", {}))
+    RE.md.update(re_metadata(iconfig))  # programmatic metadata
+
+    if subscribers:
+        for instance in subscribers:
+            if instance is None:
+                continue
+
+            # Check if it's a tiled client
+            if isinstance(instance, tiled.client.container.Container):
+                try:
+                    tiled_writer = TiledWriter(instance)
+                    RE.subscribe(tiled_writer)
+                except Exception:
+                    logger.exception(
```

Comment:
Looks like we're swallowing every exception here. Catching them to log them makes sense, but maybe re-raise after logging. Otherwise, you'll only know that your tiled writer has an problem if you look at the logs.